app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,19 @@
+<% provide(:title, "About") %>
```

Comment:
1. Xóa ngoặc ` provide(`
2. Sử dụng I18n nhé
==> Fix cho toàn bộ file html.erb nhé em